### ISSUE TITLE
The `<figcaption>` inside table's `<figure>` is upcasted to `<paragraph>`

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5-table#65]
It's either a bug or a feature... but after #51 the `<figcaption>` inside `<figure class="table">` get upcasted to a `<p>`.

It's a "bug" as this not happened before and at the same time a "feature" as the content doesn't get lost :man_shrugging:.

You can check this in manual tests: https://github.com/ckeditor/ckeditor5-table/blob/master/tests/manual/table.html#L12-L14

is rendered:
![selection_028](https://user-images.githubusercontent.com/247363/41583150-e7bda45a-73a3-11e8-8dd2-6dfd0c38a168.png)

**Comment by Reinmar** [comment]
It's fine. We don't support captioned tables yet, so the fact that this figcaption ends as a paragraph is the only thing we can do.

**Comment by oskarwrobel** [comment]
The famous part of documentation explains this :D
https://twitter.com/brandonkelly/status/1007404154753445888

**Comment by Reinmar** [comment]
I reported a separate ticket for captions: https://github.com/ckeditor/ckeditor5-table/issues/66
